scripts/sync-hf-model-repos.sh

```
@@ -65,15 +76,21 @@ declare -A FILES=(
   [segveh]="segment/meshseg_vehicle.onnx"
   [segbld]="segment/meshseg_building.onnx"
   [segcls]="segment/meshseg_category.onnx"
+  [blazeface]="mocap/face/face_detector.onnx"
+  [facemesh]="mocap/face/face_landmarks.onnx"
+  [faceblendshapes]="mocap/face/face_blendshapes.onnx"
+  [blazepose]="mocap/pose/pose_detector.onnx"
+  [poselandmarks]="mocap/pose/pose_landmarks.onnx"
 )
 
 sync_one() {
   local model=$1 repo="$OWNER/${REPOS[$model]}"
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)